bridge/client/app/_components/ktb-app-header/ktb-app-header.component.ts

```diff
@@ -107,12 +107,12 @@ export class KtbAppHeaderComponent implements OnInit, OnDestroy {
       const cliVersionsString = this.buildVersionString(this.getNewerVersions(availableCliVersions, cliVersion));
 
       if (bridgeVersionsString || cliVersionsString) {
-        const versionMessage = `New ${cliVersionsString ? ' Keptn CLI ' + cliVersionsString : ''} ${
-          cliVersionsString && bridgeVersionsString ? 'and' : ''
+        const versionMessage = `New ${cliVersionsString ? ' Keptn release ' + cliVersionsString : ''} ${
+          cliVersionsString && bridgeVersionsString ? ''
         }
                             ${
-                              bridgeVersionsString ? ' Keptn Bridge ' + bridgeVersionsString : ''
-                            } available. For details how to upgrade visit
+                              bridgeVersionsString
+                            } available. Please upgrade to this release following these instructions
                             <a href="https://keptn.sh/docs/${semver.major(latestVersion)}.${semver.minor(
           latestVersion
         )}.x/operate/upgrade/" target="_blank">
```

Comment:
Something like the following should work for the message (not tested):

```suggestion
const versionMessage = `New Keptn release ${cliVersionsString} available. <span class="bold">Please upgrade</span> to this release following these
                            <a href="https://keptn.sh/docs/${semver.major(latestVersion)}.${semver.minor(
          latestVersion
        )}.x/operate/upgrade/" target="_blank">
                            instructions</a>`;
```

Also, changing the icon might be trickier than initially toughed as it's set inside [`ktb-notification.component.html`](https://github.com/keptn/keptn/blob/103775d43c716073f47f57bf2dbbca7c99faeb8d/bridge/client/app/_components/ktb-notification/ktb-notification.component.html#L8). If you change the icon there directly it will be changed for all notifications, which I don't think is the wanted behavior.

Reply:
Okay, so not changing the icon as of now.

Reply:
Just had a look at the usage of [ktb-notification.component.html](https://github.com/keptn/keptn/blob/103775d43c716073f47f57bf2dbbca7c99faeb8d/bridge/client/app/_components/ktb-notification/ktb-notification.component.html#L8). It's only used once for the update message so you can just replace the icon.